group1/comm/static/comm_tool.js

```diff
@@ -592,22 +606,26 @@ function getCurrentRoom() {
 
 function editTeamFunc() {
   var edited_team_name = $('input#teamname').val();
-  if(testNameValidation(edited_team_name)) {
+  if(!testNameValidation(edited_team_name)) {
+    alert("Please enter a valid team name");
+
+  } else if(!isTeamNameExist(edited_team_name)){
```

Comment:
There should be a space between "if" and "(".  There should also be a space between ")" and "{" at the end of the line.